spec/ddtrace/benchmark/transport_benchmark_spec.rb

```diff
@@ -18,13 +18,25 @@
       # in a single method call. This would translate to
       # up to 1000 spans per second in a real application.
       let(:steps) { [1, 10, 100, 1000] }
-      let(:transport) { Datadog::Transport::HTTP.default }
-
-      include_examples 'benchmark'
+      let(:transport) { Datadog.tracer.writer.transport }
```

Comment:
Why initialize the tracer and skim the transport from the writer?

I think it's better to test these components in isolation. If we want to check the tracer's default transport, then we should probably benchmark this in a more holistic, black box integration test.